frameworks/g-cloud-9/questions/services/securityGovernanceStandards.yml

```diff
@@ -15,8 +15,10 @@ type: checkboxes
 options:
   - label: CSA CCM v3.0
     value: csa_ccm
+    filter_label: CSA CCM version 3.0
```

Comment:
This could probably just be changed in the label itself, so we don't need to add the filter_label key.